kokoro/ubuntu/new_release.sh

```diff
@@ -29,10 +30,14 @@ cd git/google-cloud-eclipse
 #
 # https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2363#issuecomment-327844378
 # https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/2211
-readonly CT4E_DISPLAY_VERSION=$( \
-  xmlstarlet sel -t -v '/product/@version' gcp-repo/metadata.product )
+readonly CT4E_DISPLAY_VERSION="$( \
+  xmlstarlet sel -t -v '/product/@version' gcp-repo/metadata.product )"
 readonly NO_QUALIFIER_VERSION="${CT4E_DISPLAY_VERSION%.qualifier}"
```

Comment:
What should happen if we don't have a .qualifier?  This is presumably an error. Should we error?

Reply:
We always have ".qualifier" at the end in `gcp-repo/metadata.product`. But if it wasn't there, we would already be happy, and moreover, probably it makes sense to not touch the current value in that case (which is the behavior of the current code).